config.default.json

```diff
@@ -1,3 +1,200 @@
 {
-  "BROKER_SERVER_UNIVERSAL_CONFIG_ENABLED": false
+    "BROKER_BOOT_MODE": "classic",
+    "BROKER_SERVER_URL": "https://broker.snyk.io",
+    "BROKER_HEALTHCHECK_PATH": "/healthcheck",
+    "ACCEPT_LARGE_MANIFESTS": "false",
+    "ARTIFACTORY": {
+        "BROKER_CLIENT_VALIDATION_URL":"https://$ARTIFACTORY_URL/api/system/ping",
+        "BROKER_CLIENT_VALIDATION_JSON_DISABLED":"true"
+    },
+    "AZURE-REPOS": {
+        "AZURE_REPOS_HOST":"dev.azure.com",
+        "BROKER_CLIENT_VALIDATION_BASIC_AUTH":"PAT:$AZURE_REPOS_TOKEN",
+        "BROKER_CLIENT_VALIDATION_URL":"https://$AZURE_REPOS_HOST/$AZURE_REPOS_ORG/_apis/git/repositories",
+        "GIT_URL":"$AZURE_REPOS_HOST/$AZURE_REPOS_ORG",    
+        "GIT_USERNAME":"PAT",
+        "GIT_PASSWORD":"$AZURE_REPOS_TOKEN"
+    },
+    "BITBUCKET-SERVER": {
+        "BITBUCKET_API":"$BITBUCKET/rest/api/1.0",
+        
+        "BROKER_CLIENT_VALIDATION_URL":"https://$BITBUCKET/rest/api/1.0/projects",
+        "BROKER_CLIENT_VALIDATION_BASIC_AUTH":"$BITBUCKET_USERNAME:$BITBUCKET_PASSWORD",
+        
+        "GIT_URL":"$BITBUCKET",
+        "GIT_USERNAME":"$BITBUCKET_USERNAME",
+        "GIT_PASSWORD":"$BITBUCKET_PASSWORD"
+    },
+    "CR-AGENT": {
+      "CR_AGENT_URL":"https://<agent-host>:<agent-port>"
+    },
+    "GITHUB": {
+        "GITHUB":"github.com",
+        "GITHUB_RAW":"raw.githubusercontent.com",
+        "GITHUB_API":"api.github.com",
+        "GITHUB_GRAPHQL":"api.github.com",
+
+        "BROKER_CLIENT_VALIDATION_URL":"https://$GITHUB_API/user",
+        "BROKER_CLIENT_VALIDATION_AUTHORIZATION_HEADER":"token $GITHUB_TOKEN",
+        "GIT_URL":"$GITHUB",
+        "GIT_USERNAME":"x-access-token",
+        "GIT_PASSWORD":"$GITHUB_TOKEN"
+    },
+    "GITHUB-ENTERPRISE": {
+        
```

Comment:
@aarlaud, do we need GITHUB variable here, so value substitution works as expected?

Reply:
clarified